examples/benchmark/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# run below command on M1 mac to build image for linux/amd64
+FROM --platform=linux/amd64 python:3-alpine   
+
+# Set the working directory to /app
+WORKDIR /app
+
+# Copy the current directory contents into the container at /app
+COPY standalone-data-generator/ /app
+COPY . /app
+
+# Install any needed packages specified in requirements.txt
+RUN pip install --no-cache-dir -r requirements.txt
```

Comment:
Do we have this file?

Reply:
I create and remove requirements.txt file in create-ecs-performance-test.sh.